tobrot/helper_funcs/split_large_files.py

```diff
@@ -71,9 +74,18 @@ async def split_large_files(input_file):
                 str(start_time),
                 str(end_time)
             ))
-            start_time = end_time
+            LOGGER.info(f"Start time {start_time}, End time {end_time}, Itr {i}")
+
+            #adding offset of 3 seconds to ensure smooth playback 
+            start_time = end_time-3
```

Comment:
can you add spaces before and after the operator
"end_time - 3"
?

Reply:
Yea i forgot that one

Reply:
Done